generators/client/files-react.js

```diff
@@ -404,7 +406,7 @@ const files = {
             condition: generator => generator.authenticationType !== 'oauth2',
             path: TEST_SRC_DIR,
             templates: [
-                'spec/app/modules/account/register/register.spec.tsx',
+                // 'spec/app/modules/account/register/register.spec.tsx',
```

Comment:
why is this commented out?

Reply:
if you look into source code, then, all test cases are commented in this file. During execution Jest marks this as failure with no executable test cases. I tried to convert one test case to valid, but, later during app generation it leads to some conflicts with prettier. We can create separate issue to enable test case of this class and while doing so can uncomment this

Reply:
oh, ok now I remember, I think there was some issue in this specific spec, so ya its fine